src/views/web/index.php

```diff
@@ -1,79 +1,94 @@
+                <div class="alert alert-success">
+                    <strong>Congratulations! Your server configuration satisfies all requirements.</strong>
+                </div>
+            <?php endif; ?>
 
-        <h3>Details</h3>
+            <h3>Details</h3>
 
-        <table class="table table-bordered">
-            <tr><th>Name</th><th>Result</th><th>Required By</th><th>Memo</th></tr>
-            <?php foreach ($requirements as $requirement) : ?>
-            <tr class="<?php echo $requirement['condition'] ? 'success' : ($requirement['mandatory'] ? 'danger' : 'warning') ?>">
-                <td>
-                <?php echo $requirement['name'] ?>
-                </td>
-                <td>
-                <span class="result"><?php echo $requirement['condition'] ? 'Passed' : ($requirement['mandatory'] ? 'Failed' : 'Warning') ?></span>
-                </td>
-                <td>
-                <?php echo $requirement['by'] ?>
-                </td>
-                <td>
-                <?php echo $requirement['memo'] ?>
-                </td>
-            </tr>
-            <?php endforeach; ?>
-        </table>
-    </main>
-    <hr>
-    <footer>
-        <p>Server: <?php echo $this->getServerInfo() . ' ' . $this->getNowDate() ?></p>
-        <p>Powered by <a href="https://www.yiiframework.com/" rel="external">Yii Framework</a></p>
-    </footer>
-</div>
+            <table class="table table-bordered">
+                <tr>
+                    <th>Name</th>
+                    <th class="text-center">Result</th>
+                    <th>Required By</th>
+                    <th>Note</th>
+                </tr>
+                <?php foreach ($requirements as $requirement) : ?>
+                    <tr class="<?php echo $requirement['condition'] ? 'success' : ($requirement['mandatory'] ? 'danger' : 'warning') ?>">
+                        <td>
+                            <?php echo $requirement['name'] ?>
+                        </td>
+                        <td class="text-center">
+                            <span class="result">
+                                <!-- <span class="result">< ?php echo $requirement['condition'] ? 'Passed' : ($requirement['mandatory'] ? 'Failed' : 'Warning') ?></span> -->
+                                <?php echo $requirement['condition'] ? '<img src="yii3_sign.svg" height="20" alt="Passed">' : ($requirement['mandatory'] ? '<img src="yii3_sign_black.svg" height="20" alt="Failed">' : '<img src="yii3_sign_grey.svg" height="20" alt="Warning">') ?>
+                            </span>
+                        </td>
+                        <td>
+                            <?php echo $requirement['by'] ?>
+                        </td>
+                        <td>
+                            <?php echo $requirement['memo'] ?>
+                        </td>
+                    </tr>
+                <?php endforeach; ?>
+            </table>
+        </main>
+        <hr>
```

Comment:
Overall `<?=` and escaping of output are needed.

Reply:
?